releases/release-1.22/release-notes/maps/pr-100731-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 100731
+releasenote:
+  text: 'azurefile: Normalize share name to not include the capital letters'
```

Comment:
Any reason why this is single quotes ? Just curious

Reply:
> Any reason why this is single quotes ? Just curious

Hmm, not really sure and they don't make any sense too. Do you think we should remove them ?

Reply:
You need them because YAML will complain about having `:`